x-pack/metricbeat/module/aws/_meta/kibana/7/visualization/be8828d0-f7f6-11e8-af03-c999c9dea608-ecs.json

```diff
@@ -54,7 +54,7 @@
                         "steps": 0,
                         "terms_field": "cloud.instance.id",
                         "terms_order_by": "61ca57f2-469d-11e7-af02-69e470af7417",
-                        "terms_size": "5"
+                        "terms_size": "10"
```

Comment:
I changed this from 5 to 10 to match all other visualizations in this dashboard.